lib/manager/components/version/FeedVersionReport.js

```diff
@@ -74,31 +74,31 @@ export default class FeedVersionReport extends Component<Props, State> {
       <Panel
         // bsStyle='info'
       >
-        <Panel.Heading>
+        <Panel.Heading><Panel.Title componentClass='h3'>
```

Comment:
Not sure if need the h3 here, there is a subheading underneath.

Reply:
Yep good call thanks. Fixed in 566bef2e3b25f7411ef8ad1d0183d5763c185b39